ConsoleApp/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+bool valid(string val)
```

Comment:
method name not correct, should be
```suggestion
bool Valid(string val)
```